### DISCUSSION TITLE
Customize fan curve levels

### DISCUSSION THREAD

**Discussion by omri09** [BartoszCichecki/LenovoLegionToolkit#1499] [Q&A] [answered]
Hello Guys,

I want to be able to customize the fan curve levels. For example, there is a step for an 85°C CPU, and the next step is 94°C.
Is there a way to control these temperature levels?

Thanks.

**Comment by Ace-Radom** [comment, marked as answer]
@omri09 Please make yourself clear with README. Fan curve customize is available in custom mode if you have a gen7 or later models. If you have older models, it's not possible due to technical limitations.
